webapp/pages/profile/_id/_slug.spec.js

```diff
@@ -21,12 +21,15 @@ describe('ProfileSlug', () => {
         id: 'p23',
         name: 'It is a post',
       },
-      $t: jest.fn(),
+      $t: jest.fn(a => a),
```

Comment:
:+1: that way you can test translations :point_down: in `wrapper.text()`

Reply:
Yeahhhh ;-)